account_financial_report/wizard/general_ledger_wizard_view.xml

```diff
@@ -91,6 +91,14 @@
                                 options="{'no_create': True}"
                             />
                         </page>
+                        <page string="Filter journals">
+                            <field
```

Comment:
Since many companies have lots of journals : please make the field span the page (like the account_ids in the first notebook page.

![image](https://user-images.githubusercontent.com/7706945/197503289-68673c6d-d809-4552-a4f2-1ef88838e54a.png)

Reply:
The view is changed and with the same layout as the account filtering